src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/pool-edit-peer-modal/pool-edit-peer-modal.component.html

```diff
@@ -0,0 +1,113 @@
+<cd-modal>
+  <ng-container class="modal-title"
+                i18n>{mode, select, edit {Edit} other {Add}}
+  pool mirror peer</ng-container>
+
+  <ng-container class="modal-content">
+    <form name="editPeerForm"
+          class="form"
+          #formDir="ngForm"
+          [formGroup]="editPeerForm"
+          novalidate>
+      <div class="modal-body">
+        <p>
+          <ng-container i18n>{mode, select, edit {Edit} other {Add}} the pool
+          mirror peer attributes for pool <kbd>{{ poolName }}</kbd> and click 
+          <kbd>Submit</kbd>.</ng-container>
+        </p>
+
+        <div class="form-group"
+             [ngClass]="{'has-error': editPeerForm.showError('clusterName', formDir)}">
+          <label class="control-label"
+                 for="clusterName">
+            <span i18n>Cluster Name</span>
```

Comment:
nit: Required fields must have a `*` next to the label (use `<span class="required"></span>`):

![screenshot from 2018-12-16 22-41-19](https://user-images.githubusercontent.com/14297426/50059910-f8068b00-0184-11e9-9339-c5f3815894d8.png)